src/hiral_purohit/LastConsonant.java

```diff
@@ -0,0 +1,20 @@
+package hiral_purohit;
+
+public class LastConsonant {
+
+	void printConsonant(String str) {
+		char lastCon = ' ';
+		for (int i = 0; i < str.length(); i++) {
+			char ch = str.charAt(i);
+			if (ch != 'a' && ch != 'e' && ch != 'i' && ch != 'o' && ch != 'u') {
```

Comment:
You can use OR condition

Reply:
i have tried or condition also and that is also working.